zns/seat-capsule.c

```diff
@@ -2,13 +2,36 @@
 
 #include <cglm/quat.h>
 #include <cglm/vec3.h>
+#include <math.h>
 #include <zen-common.h>
 
 #include "board.h"
 #include "bounded.h"
 #include "zen/appearance/board.h"
 #include "zen/virtual-object.h"
 
+#define INITIAL_BOARD_GAP (M_PI / 10.f)
+
+void
+zns_seat_capsule_rearrange(struct zns_seat_capsule *self)
+{
+  int board_len = wl_list_length(&self->board_list);
+  int i = 0;
+  float prev_azim;
+  struct zns_board *board;
+  wl_list_for_each (board, &self->board_list, seat_capsule_link) {
+    float azim;
```

Comment:
```suggestion
    float azimuthal;
```

We do not prefer abbreviations unless they are widely accepted.
The same applies to `prev_azim`.

Although "board_len" is probably acceptable, I would use "board_length" because I don't want to think about whether I can use that abbreviation.